rpcs3/Emu/VFS.cpp

```diff
@@ -689,6 +707,18 @@ std::string vfs::unescape(std::string_view name)
 			}
 			break;
 		}
+		case char2{u8" "[0]}:
```

Comment:
I'd like to use different escape character for space, something readable but unused from https://en.wikipedia.org/wiki/Halfwidth_and_Fullwidth_Forms_(Unicode_block)

Reply:
I think ＿ fits.

Reply:
Fixed.